chain/client/src/test_utils/test_env_builder.rs

```diff
@@ -280,7 +284,8 @@ impl TestEnvBuilder {
     /// Visible for extension methods in integration-tests.
     pub fn internal_ensure_epoch_managers_for_nightshade_runtime(
         self,
-    ) -> (Self, Vec<PathBuf>, Vec<Store>, Vec<Arc<EpochManagerHandle>>) {
+    ) -> (Self, Vec<PathBuf>, Vec<Store>, Vec<Arc<EpochManagerHandle>>, bool) {
```

Comment:
The return type deserves either to be given a name or a comment. I would prefer the former but either is fine.

Reply:
I'll follow up on this in a separate PR. This can be further improved IMO.